cognite_toolkit/modules/cdf_data_pipeline_asset_default/datasets/dataset.yaml

```diff
@@ -0,0 +1,20 @@
+external_id: 'ds-asset:{{location_name}}'
```

Comment:
In the group, you only have assetsAcl, but below in the metadata you refer to raw and transformations? How do we maintain the integrity of the metadata? 
And shouldn't we use writeProtection and the dataSetOwner capability?